src/bun.js/test/jest.zig

```diff
@@ -1705,3 +1748,180 @@ pub fn printGithubAnnotation(exception: *JSC.ZigException) void {
     Output.printError("\n", .{});
     Output.flush();
 }
+
+pub const EachData = struct { strong: JSC.Strong, is_test: bool };
+
+fn eachBind(
+    globalThis: *JSGlobalObject,
+    callframe: *CallFrame,
+) callconv(.C) JSValue {
+    comptime var signature = "eachBind";
+    const callee = callframe.callee();
+    const arguments = callframe.arguments(3);
+    const args = arguments.ptr[0..arguments.len];
+
+    if (args.len < 2) {
+        globalThis.throwPretty("{s} a description and callback function", .{signature});
+        return .zero;
+    }
+
+    var description = args[0];
+    var function = args[1];
```

Comment:
where does it check if `function` is a function? You can use `.isCallable` for that.